### ISSUE TITLE
OpenSSL Source Outdated

### ISSUE THREAD

**Issue by tiandavis** [rvm/rvm#3417]
Running `rvm install 2.1.1` fails because the OpenSSL source URL is outdated.

Here's the rvm output:

```
==> Upgrading 1 outdated package, with result:
openssl 1.0.2
==> Upgrading openssl
==> Downloading https://www.openssl.org/source/openssl-1.0.2.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Trying a mirror...
==> Downloading https://raw.githubusercontent.com/DomT4/LibreMirror/master/OpenSSL/openssl-1.0.2.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "openssl"
Download failed: https://raw.githubusercontent.com/DomT4/LibreMirror/master/OpenSSL/openssl-1.0.2.tar.gz
Requirements installation failed with status: 1.
```

We're getting a 404 for:
https://www.openssl.org/source/openssl-1.0.2.tar.gz

Also getting a 404 for the backup source:
https://raw.githubusercontent.com/DomT4/LibreMirror/master/OpenSSL/openssl-1.0.2.tar.gz

The updated recipe should probably use:
https://www.openssl.org/source/openssl-1.0.2-latest.tar.gz

Currently installing Ruby 2.1.\* with rvm simply does not work at all. Any help with this would be great. Thanks Everyone!

**Comment by ojab** [comment]
What rvm version and what OS do you use? I assume that it's Mac OS X with brew installed, looks like it's the issue with brew, not with rvm.
Try `brew update && brew upgrade --all` and `rvm install 2.1.1` after that.

**Comment by wvega** [comment]
I ran into the same issue trying to execute `rvm install 2.2.3` with RVM 1.26.10.

@ojab's suggestion worked for me on Mac OS X 10.11 (I also updated RVM to 1.26.11).

**Comment by ram535ii** [comment]
Was in the exact same position and @wvega and @ojab's suggestion also fixed it for me.

**Comment by BoBao77** [comment]
Was upgrading to 2.3.0 and ran into the same issues. The above suggestions worked for me too.
RVM 1.27.0, MAC OS X 10.11.1 

At first my current brew was all jacked up though, and failed during the update. To get it functional again I had to reference this http://stackoverflow.com/questions/24652996/homebrew-not-working-on-osx

and then I had to sudo brew update - hopefully no ones setup is jacked up like mine
